modules/objdetect/include/opencv2/objdetect/aruco_detector.hpp

```diff
@@ -10,7 +10,8 @@
 namespace cv {
 namespace aruco {
 
-/** @defgroup aruco ArUco Marker Detection
+/** @defgroup objdetect_aruco
+ * ArUco Marker Detection
```

Comment:
```
/build/precommit_docs/4.x/opencv/modules/objdetect/include/opencv2/objdetect/aruco_detector.hpp:13: warning: missing title after \defgroup objdetect_aruco
```

Reply:
See how it looks in final docs: https://pullrequest.opencv.org/buildbot/export/pr/22986/docs/d5/d54/group__objdetect.html

Reply:
Now this looks like this:
![image](https://user-images.githubusercontent.com/22337800/209177999-d9736984-3dee-436d-8ff6-ef1c8d3f89a2.png)
Is it ok?
https://pullrequest.opencv.org/buildbot/export/pr/22986/docs/d5/d54/group__objdetect.html

Reply:
No dots in headers

Reply:
"ArUco" will better

Reply:
fixed